src/Symfony/Component/Intl/Intl.php

```diff
@@ -54,15 +54,8 @@ final class Intl
      */
     public const TIMEZONE_DIR = 'timezones';
 
-    /**
-     * @var string|bool|null
-     */
-    private static $icuVersion = false;
-
-    /**
-     * @var string
-     */
-    private static $icuDataVersion = false;
+    private static string|bool|null $icuVersion = false;
```

Comment:
should it be `string|false|null` instead ? We never put `true` in it.

Reply:
b2a1d8c74af677c911cff5fb8fe78abaf5fed60c